raiden_contracts/tests/test_channel_unlock.py

```diff
@@ -57,16 +57,66 @@ def test_merkle_root_1_item_unlockable(
     assert unlocked_amount == 6
 
 
-def test_merkle_root(
+def test_merkle_tree_length_fail(
+        web3,
+        get_accounts,
+        token_network_test_utils,
+        secret_registry_contract,
+):
+    network_utils = token_network_test_utils
+    A = get_accounts(1)[0]
+    pending_transfers_tree = get_pending_transfers_tree(web3, [2, 3, 6], [5])
+
+    secret_registry_contract.functions.registerSecret(
+        pending_transfers_tree.unlockable[0][3],
```

Comment:
Can you make clearer what the `[0][3]` refers to?

Reply:
Done, I added a `TestLockIndex` enum to make it explicit.